tabbycat/adjfeedback/views.py

```diff
@@ -704,6 +706,31 @@ class PublicFeedbackProgress(PublicTournamentPageMixin, BaseFeedbackProgressView
     public_page_preference = 'feedback_progress'
 
 
+class IgnoreFeedbackView(SingleObjectFromTournamentMixin, RedirectView):
+    model = AdjudicatorFeedback
+    pk_url_kwarg = 'feedback_id'
+    allow_null_tournament = False
+    private_url = True
+    tournament_field_name = 'adjudicator__tournament'
+
+    def get_redirect_url(self, *args, **kwargs):
```

Comment:
Just a couple of pointers for future use:

By convention, HTTP GET requests aren't meant to be used for data-changing actions, in principle. We're meant to use HTTP POST (or PUT) requests for those instead. This way, browsers will know not to cache them, allow them to be bookmarked, or store them in the user's history. We've made this mistake before ourselves, and actually once had to convert all our data-changing GETs to POSTs (see issue #287).

The flipside is that writing a web page (on the front-end) to send a POST request can be nontrivial relative to a GET request—by design, you can't just use a standard `<a>` tag for a POST request (they're always GETs). We actually haven't decided on a standard pattern for how we do POST requests yet, so we'll take care of it after merge—no need for action your side. I just wanted to give you a heads-up about the convention.

The other note is that I try to write it so that `get_redirect_url()` does only what it says, construct and return a redirect URL. The actual data-changing action should be in other methods—for form views there's a structure for this, but for something like this I just put it in `post()` and then call the super at the end of the `post()` method. There's a `PostOnlyRedirectView` for this purpose which provides a standard pattern for the back-end (but not the front-end). Again, don't worry about this, we'll deal with it after merging, once we've settled on a pattern for the front-end.

Reply:
I know its more of an anti-pattern, but a proper button would seem out-of-place.

Reply:
Yeah, this is why we weren't decided on the pattern 🙂. There are ways to do it with a link, but they involve fiddling with ajax or something like that (@philipbelesky knows more than I do).